src/pages/_app.tsx

```diff
@@ -143,7 +143,7 @@ const MyApp: FC<AppProps> = ({Component, pageProps}) => {
   return (
     <>
       <Head>
-        <title>React App</title>
+        <title>STROM</title>
```

Comment:
default asi treba. ale kedy sa vlastne este zobrazuje? v adminovi by sme asi mali pridat nieco tiez eventualne

Reply:
zobrazuje sa to ked si na Razcestniku napriklad

Reply:
akoze teoreticky mozeme mat pre React admin nieco vlastne, ale STROM moze byt vsade :-D